### PR TITLE
Allow use of Organizational trail to be toggled via variable

### PR DESCRIPTION
**Pull request by jamiedick**

### CODE REVIEW

**Comment by jamiedick** [comment]
@nozaq please let me know any feedback you have

File: main.tf [hunk 1]
```diff
@@ -81,7 +81,7 @@ module "cloudtrail_baseline" {
   s3_object_level_logging_buckets   = var.cloudtrail_s3_object_level_logging_buckets
   dynamodb_event_logging_tables     = var.cloudtrail_dynamodb_event_logging_tables
   lambda_invocation_logging_lambdas = var.cloudtrail_lambda_invocation_logging_lambdas
-  is_organization_trail             = local.is_master_account
+  is_organization_trail             = var.is_organization_trail
```
Comment (nozaq) [inline main.tf:84]: Thank you for opening the PR, and sorry for the delay in responding!

I agree that adding a way to toggle the usage of the organizational trail would be beneficial.
One suggestion is to use the organization trail for the master account by default for backward compatibility.

One possible implementation is to change `var.is_organization_trail` to `var.turn_off_organization_trail` with the default value `false`, then use `local.is_master_account && !var.turn_off_organization_trail`.

In this case, the default behavior would be to turn on the organizational trail for the master account and turn off for member accounts by default, which is backward-compatible.

How do you think?

Comment (jamiedick) [reply main.tf:84]: That makes sense to me - i'll make the change shortly.  Thanks for the response!

**Comment by jamiedick** [comment]
@nozaq unrelated - but I believe the terraform validate is failing bc of a new/updated s3 bucket module.  I ran into something similar earlier this week.  Let me know your thoughts

**Review by nozaq — COMMENTED** [review]
Thanks for the quick update!

> but I believe the terraform validate is failing bc of a new/updated s3 bucket module. I ran into something similar earlier this week.

Yeah, it looks like so. AWS provider has some breaking changes in v4.0. We can ignore the validation failure this time 👍🏼

File: main.tf [hunk 2]
```diff
@@ -30,6 +30,7 @@ locals {
   is_individual_account = var.account_type == "individual"
   is_master_account     = var.account_type == "master"
   is_cloudtrail_enabled = var.cloudtrail_baseline_enabled && (local.is_individual_account || local.is_master_account)
+  is_organization_trail = var.turn_off_organization_trail ? false : true
```
[the remark this answers is not in the document]
Comment (jamiedick) [reply main.tf]: done!

**Comment by nozaq** [comment]
@jamiedick LGTM, thanks for the contribution!

**Pull request merged**